terminal_check_appengine.go

```diff
@@ -1,4 +1,4 @@
-// +build appengine gopherjs
+// +build appengine js
```

Comment:
Do we still need the js build constraint here if we have another file dedicated for the gopherjs environment ?